[MTM-54103] document pick from existing data point option (#1176)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/data-point-library.md

|Yellow range|Min/max values for the yellow range (MINOR alarms)
|Red range|Min/max values for the red range (CRITICAL alarms)

{{< c8y-admon-info >}}

> The button next to the fragment and series fields allows the user to pick the values for fragment and series from an existing data point using the data point selector.

Reviewer: ```suggestion
With the button next to the fragment and series fields you can pick the values for fragment and series from an existing data point using the data point selector.
```
